ui/components/app/confirmation/confirm-footer/confirm-footer.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
Please following naming convention as in other confirmation components here: https://github.com/MetaMask/metamask-extension/pull/22081